Cargo.toml

```diff
@@ -22,18 +22,18 @@ maintenance = { status = "experimental" }
 
 [dependencies]
 futures = "0.3"
-tokio = { version = "1.22", features = ["net", "rt", "time"] }
+tokio = { version = "1.37", features = ["net", "rt", "time"] }
 failure = "0.1"
-byteorder = "1.2"
-slog = "2.3.2"
-async-trait = "0.1.58"
-pin-project = "1.0.12"
-once_cell = "1.17.0"
+byteorder = "1.5"
+slog = "2.7"
+async-trait = "0.1"
+pin-project = "1.1"
+once_cell = "1.19"
 #slog = { version = "2.3.2", features = ['max_level_trace'] }
```

Comment:
We can probably drop slog while we're at it.

Reply:
17fbe78cefd1bdfd97d3c4eeb4ac1090337b31d0

Reply:
Fwiw, we already didn't use slog, this is just an old leftover.

Reply:
Not sure what you mean by "we already didn't use slog", I mean we had a dependency on slog in line 28. It was used across the code base, I understood you comment to actually get rid of it (and I replaced it with tracing)

Reply:
Aargh, you're right. I'm dumb. I missed that we had both a commented-out and uncommented slog entry in the dependency list.

Going back, I'd say that the migration would've been out of scope for this PR, but I'm still +1 on doing it, and since the work is already done now..